### ISSUE TITLE
`no-magic-numbers` should ignore parseInt radix param

### ISSUE THREAD

**Issue by nekoya** [palantir/tslint#3514]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.5.3

#### TypeScript code being linted

```ts
parseInt('123.4', 10);
```

with `tslint.json` configuration:

```json
"no-magic-numbers": true,
"radix": true
```

#### Actual behavior

```
ERROR: src/parseint.ts[1, 19]: 'magic numbers' are not allowed
```

#### Expected behavior

No errors. I don't think this value is Magical.

ESLint ignores radix value for `parseInt`.

- https://github.com/eslint/eslint/issues/4167

I'm understanding that we can specify allowed numbers like below.

```
"no-magic-numbers": [true, 10]
```

But I'd like to catch a warning when the value is used not in parseInt.

**Comment by ajafff** [comment]
Fixed by #3536

**Comment by m-przybylski** [comment]
I'm using tslint in version 5.11.0 and this issue still occurs in Number.parseInt function (parseInt from Number namespace), and does not occur for parseInt from global scope.
